_includes/header.html

```diff
@@ -51,22 +54,16 @@
                         </a>
                         <ul class="dropdown-menu">
                             <li>
-                                <a href="/projects" title="Discover projects that use ev3dev">Projects</a>
-                            </li>
-                            <li>
-                                <a href="/share" title="Share projects that use ev3dev">Share</a>
+                                <a href="/projects" title="Discover projects that use ev3dev">Project showcase</a>
```

Comment:
For some reason, I feel like this should be the last item on the list (and Contribute should be first). Just intuition I gues.